license/licenses.py

```diff
@@ -122,3 +122,33 @@ class AGPLv3OnlyLicense(AGPLv3LaterLicense):
     id = 'AGPL-3.0'
     rpm = 'AGPLv3'
     python = 'License :: OSI Approved :: GNU Affero General Public License v3'
+
+
+class Apachev1License(License):
+    '''
+    Apache License Version 1.0
+    '''
+    id = 'Apache-1.0'
+    rpm = 'ASL 1.0'
+    python = 'License :: OSI Approved :: Apache Software License'
+    url = 'https://www.apache.org/licenses/LICENSE-1.0'
+
+
+class Apachev11License(License):
+    '''
+    Apache License Version 1.1
+    '''
+    id = 'Apache-1.1'
+    rpm = 'ASL 1.1'
+    python = 'License :: OSI Approved :: Apache Software License'
```

Comment:
Thanks for pointing that, I was confused by *LaterLicense and *OnlyLicense classes and have not notice the BSD3ClauseLicense design.

I will change it immediately.